<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HistoryWindow</class>
 <widget class="QWidget" name="HistoryWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>399</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>540</width>
    <height>310</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>History</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../res/res.qrc">
    <normaloff>:/icons/history.ico</normaloff>:/icons/history.ico</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="1,100" columnminimumwidth="200,0">
   <property name="margin">
    <number>6</number>
   </property>
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="refreshButton">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Refresh</string>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="exportButton">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Export</string>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="importButton">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Import</string>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="printButton">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Print</string>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="filtersButton">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Filters</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="searchButton">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Search</string>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="searchEdit">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="placeholderText">
        <string>Quick search</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="contactSearchEdit">
       <property name="placeholderText">
        <string>Search contacts</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTreeView" name="contactTree">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="indentation">
        <number>10</number>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="headerDefaultSectionSize">
        <number>0</number>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="filtersWidget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QGroupBox" name="filtersGroupBox">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="title">
           <string>Filters</string>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <property name="leftMargin">
            <number>6</number>
           </property>
           <property name="topMargin">
            <number>3</number>
           </property>
           <property name="rightMargin">
            <number>6</number>
           </property>
           <property name="bottomMargin">
            <number>6</number>
           </property>
           <item row="1" column="0">
            <widget class="QLabel" name="dateFromLabel">
             <property name="text">
              <string>From:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QDateEdit" name="dateFromEdit">
             <property name="date">
              <date>
               <year>1970</year>
               <month>1</month>
               <day>1</day>
              </date>
             </property>
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QDateEdit" name="dateToEdit">
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="dateToLabel">
             <property name="text">
              <string>To:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="2">
            <widget class="QComboBox" name="directionComboBox">
             <item>
              <property name="text">
               <string>Both directions</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Incoming</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Outgoing</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QTreeWidget" name="chatTree">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>100</height>
       </size>
      </property>
      <property name="indentation">
       <number>0</number>
      </property>
      <column>
       <property name="text">
        <string notr="true">Contact</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Duration</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Count</string>
       </property>
      </column>
     </widget>
     <widget class="Kitty::ChatWebView" name="chatWebView">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
  <customwidget>
   <class>Kitty::ChatWebView</class>
   <extends>QWebView</extends>
   <header>widgets/ChatWebView.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>chatTree</tabstop>
  <tabstop>contactSearchEdit</tabstop>
  <tabstop>searchEdit</tabstop>
  <tabstop>directionComboBox</tabstop>
  <tabstop>dateFromEdit</tabstop>
  <tabstop>dateToEdit</tabstop>
  <tabstop>refreshButton</tabstop>
  <tabstop>exportButton</tabstop>
  <tabstop>importButton</tabstop>
  <tabstop>printButton</tabstop>
  <tabstop>filtersButton</tabstop>
  <tabstop>searchButton</tabstop>
  <tabstop>contactTree</tabstop>
  <tabstop>chatWebView</tabstop>
 </tabstops>
 <resources>
  <include location="../../../res/res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>filtersButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>filtersWidget</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>89</x>
     <y>366</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dateFromEdit</sender>
   <signal>dateChanged(QDate)</signal>
   <receiver>HistoryWindow</receiver>
   <slot>updateCurrentChat()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>306</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dateToEdit</sender>
   <signal>dateChanged(QDate)</signal>
   <receiver>HistoryWindow</receiver>
   <slot>updateCurrentChat()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>348</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>directionComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>HistoryWindow</receiver>
   <slot>updateCurrentChat()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>103</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>searchEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>HistoryWindow</receiver>
   <slot>updateCurrentChat()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>539</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updateCurrentChat()</slot>
 </slots>
</ui>
